doc-content/enterprise-only/admin-and-config/business-central-editing-data-sources-proc.adoc

```diff
@@ -0,0 +1,70 @@
+[id='managing-business-central-data-sources-con']
+= Managing data sources
+
+{CENTRAL} provides data source management which gives you the ability of defining data sources for accessing a database. These data sources are then used by other {CENTRAL} components such as the data sets. A database driver, on the other hand, is used for enabling communication between a data source and the targeted database.
+
+From the *Data Source Authoring* page you can add data sources and database drivers to {CENTRAL}.
```

Comment:
Missing a mention to AppServer data source that can't be Edited nor Deleted, but can be used

Reply:
Added note regarding the default data source.